src/components/app/App.tsx

```diff
@@ -14,6 +15,7 @@ const App: React.FC = () => {
   return (
     <Container fluid>
       <Navbar bg="primary" variant="dark" className="mb-3">
+        <HouseDoorFill color="white" size={25} />
```

Comment:
We should add some right margin to this icon so that there's a little more space between the icon and the "Elephant Vending Machine" text.